samples/assistant/README.md

```diff
@@ -38,7 +45,7 @@ The assistant will invoke a skill function whenever it decides to do so to satis
 
 The sample is available in the following language stacks:
 
-* [C# on the in-process worker](csharp-inproc)
+* [C# on the out-of-process worker](https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide?tabs=windows)
```

Comment:
The purpose of this link was to navigate the user directly to the sample. Were you intentionally changing it to point to the .NET Isolated guide or should we instead just update the link to point to the new sample location?

```suggestion
* [C# on the out-of-process worker](csharp-ooproc)
```

Reply:
ohhh I see.  I was confused what that was supposed to point to, so I had it point to the .NET isolated guide. I'll update this to point to the sample.